src/components/__tests__/LoginButtons.cy.js

```diff
@@ -0,0 +1,81 @@
+import LoginButtons from 'components/LoginButtons.vue';
+import { i18n } from '../../boot/i18n';
+
+const rideToWorkByBikeConfig = JSON.parse(
+  process.env.RIDE_TO_WORK_BY_BIKE_CONFIG,
+);
+const colorPrimary = rideToWorkByBikeConfig.colorPrimary;
+
+describe('<LoginButtons>', () => {
+  it('has translation for all strings', () => {
+    cy.testLanguageStringsInContext(
+      ['buttonGoogle', 'buttonFacebook'],
+      'login.buttons',
+      i18n,
+    );
+  });
+
+  context('desktop', () => {
+    beforeEach(() => {
+      cy.mount(LoginButtons, {
+        props: {},
+      });
+      cy.viewport('macbook-16');
+    });
+
+    it('renders login button google', () => {
+      cy.dataCy('login-button-google')
+        .should('be.visible')
+        .should('have.css', 'font-size', '14px')
```

Comment:
Why not use `.and` instead of `.should`?

Reply:
Originally for consistency. Replacing with `.and` for better readability.